fineract-provider/src/main/resources/application.properties

```diff
@@ -142,6 +142,10 @@ fineract.loan.transactionprocessor.due-penalty-interest-principal-fee-in-advance
 fineract.loan.transactionprocessor.advanced-payment-strategy.enabled=${FINERACT_LOAN_TRANSACTIONPROCESSOR_ADVANCED_PAYMENT_STRATEGY_ENABLED:true}
 fineract.loan.transactionprocessor.error-not-found-fail=${FINERACT_LOAN_TRANSACTIONPROCESSOR_ERROR_NOT_FOUND_FAIL:true}
 
+# Comma separated list of loan statuses which will be recorded on change. There are two extra values: "NONE" and "ALL".
+# "NONE" disables the feature and no entries will be created, "ALL" enables the feature for all loan statuses.
+fineract.loan.status-change-history-statuses=${FINERACT_LOAN_STATUS_CHANGE_HISTORY_STATUSES:OVERPAID}
```

Comment:
Didn't we agree that NONE is the default?

Reply:
You are absolutely right! Fixing.